pages/docs/user-docs/build-shub.md

```diff
@@ -0,0 +1,28 @@
+---
+title: shub bootstrap module
+sidebar: user_docs
+permalink: build-shub
+folder: docs
+toc: false
+---
+
+This module allows you to build a container from a container hosted on Singularity Hub. 
+
+{% include toc.html %}
+
+## Overview
+You can use an existing container on Singularity Hub as your "base," and then add customization. This allows you to build multiple images from the same starting point. For example, you may want to build several containers with the same custom python installation, the same custom compiler toolchain, or the same base MPI installation. Instead of building these from scratch each time, you could create a base container on Singularity Hub and then build new containers from that existing base container adding customizations in `%post`, `%environment`, `%runscript`, etc.
+
+## Keywords
+```
+Bootstrap: shub
+```
+The **Bootstrap** keyword is always mandatory. It describes the bootstrap module to use.
+```
+From: shub://username/container:branch
+From: shub://2976
+```
```

Comment:
oh no this is old version we now have:

```
shub://<registry>/<username>/<container-name>:<tag>@digest
```
and digest is a commit, for now :)

We also no longer have support for numbers - I removed it a while ago.

Reply:
What does registry default to?

Reply:
singularity-hub.org

Reply:
And what does `digest` default to?

Reply:
digest doesn't default to anything - it's akin to asking for a specific commit or version. If you don't specify it, you get the last modified that matches the username / container / tag.